addons/mail/static/src/core/common/chat_bubble.xml

```diff
@@ -38,11 +38,10 @@
         <t t-if="message.isSelfAuthored">
             <i class="fa fa-mail-reply me-1"/>You:
         </t>
-        <t t-elif="message.author and !message.author?.eq(thread.correspondent?.persona)">
-            <t t-esc="message.author.name"/>:
+        <t t-elif="message.author and thread.correspondent and message.author?.eq(thread.correspondent.persona)">
         </t>
```

Comment:
Can't we self close the previous tag?

I don't really get why we should change this condition? Won't this be enough?
```js
message.author?.eq(thread.correspondent?.persona)
```

Currently, if message author or thread correspondent is falsy, the condition will not be met. The same holds for the proposed change but is more concise?

Reply:
Indeed this is simpler, I was overthinking it!